src/storage/src/hummock/local_version_manager.rs

```diff
@@ -77,40 +142,38 @@ impl ScopedLocalVersion {
 /// during the lifetime of `ScopedLocalVersion`. Internally `LocalVersionManager` will pin/unpin the
 /// versions in storage service.
 pub struct LocalVersionManager {
-    current_version: RwLock<Option<Arc<ScopedLocalVersion>>>,
+    current_version: RwLock<Option<LocalVerion>>,
 
-    update_notifier_tx: tokio::sync::watch::Sender<HummockVersionId>,
-    unpin_worker_tx: UnboundedSender<Arc<HummockVersion>>,
-    unpin_worker_rx: Mutex<Option<UnboundedReceiver<Arc<HummockVersion>>>>,
-
-    /// Track the refcnt for committed epoch to facilitate shared buffer cleanup
-    committed_epoch_refcnts: Mutex<BTreeMap<u64, u64>>,
-}
+    /// Manager for immutable shared buffers
+    shared_buffer_manager: SharedBufferManager,
```

Comment:
It seems that to ensure consistency of `shared_buffer_manager` and `current_version`, we must acquire lock of `current_version` and then can do any operation on `shared_buffer_manager`. And if the operation on `shared_buffer_manager` will acquire the write lock for btreemap, we also need to acquire write lock for current_version at first. 
Is it right?